homeassistant/loader.py

```diff
@@ -35,11 +35,16 @@
 from .generated.zeroconf import HOMEKIT, ZEROCONF
 from .util.json import JSON_DECODE_EXCEPTIONS, json_loads
 
-# Typing imports that create a circular dependency
 if TYPE_CHECKING:
+    # Typing imports that create a circular dependency
+    # End typing imports that create a circular dependency
```

Comment:
Uh?

Reply:
Looks like ruff reformatted this on me and I didn't notice

Reply:
Sorry about that.  all fixed now